### ISSUE TITLE
Camera requirements in SDK Manifest are mandatory

### ISSUE THREAD

**Issue by isles1217** [klarna/klarna-mobile-sdk#107]
**Describe the bug**
When preparing a release for OSM on Android, I discovered that the Klarna SDK declares a camera requirement that differs from our app’s requirements that is NOT mentioned in the documentation. Is there a way to make these features optional in your SDK? Releasing our app with the Klarna SDK as is will result in dropping support for devices without cameras.

From the Klarna SDK’s AndroidManifest.xml file:
```
    <uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.FLASHLIGHT" />
    <uses-feature android:name="android.hardware.camera" />
    <uses-feature android:name="android.hardware.camera.autofocus" />
```

**Expected behavior**
These camera requirements can be made optional with a `required=false` tag, ie:
```
    <uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.FLASHLIGHT" />
    <uses-feature android:name="android.hardware.camera" android:required="false" />
    <uses-feature android:name="android.hardware.camera.autofocus" android:required="false" />
    (something similar for the flashlight permission)
```

**Screenshots**
![Screen Shot 2021-07-20 at 12 19 32 PM](https://user-images.githubusercontent.com/13743650/126383118-9f126bc9-895f-4139-903e-de3c6642cc20.png)

**Device and version:**
 - Device: Android
 - Klarna In-App SDK version: 2.0.42

**Merchant Name**
Babylist

**Impact Level**
Blocking release, dropping supported devices

**Additional context**

**Comment by isles1217** [comment]
Thanks a lot, @NMGuner !

**Comment by NMGuner** [comment]
Happy to notify that we released a new version(2.0.44) with optional requirements and checks in the SDK to not run into issues during runtime. I will mark this issue as resolved but feel free to update us if you keep experiencing this problem.
